<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePswd</class>
 <widget class="QWidget" name="ChangePswd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangePswd</string>
  </property>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>原密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OriginalPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>70</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #f7f9fa;
border-radius: 4px;
padding-left: 2px;
font-size: 14px;
color: rgba(0,0,0,0.3);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>120</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #f7f9fa;
border-radius: 4px;
padding-left: 2px;
font-size: 14px;
color: rgba(0,0,0,0.3);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ConfirmPasswordLineEdit">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>160</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #f7f9fa;
border-radius: 4px;
padding-left: 2px;
font-size: 14px;
color: rgba(0,0,0,0.3);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmChangePushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
border: 1px solid rgba(0,0,0,0.3);
border-radius: 4px;
color: rgba(0,0,0,0.5);
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="info_comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #e9eef2;
border-radius: 4px;
padding-left: 3px;
font-size: 14px;
color: rgba(0,0,0,0.8);
border: 0px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择身份：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ConfirmChangePushButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangePswd</receiver>
   <slot>confirmChangePswd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>confirmChangePswd()</slot>
 </slots>
</ui>
